### ISSUE TITLE
Expose `liftReducer` and/or `liftState`

### ISSUE THREAD

**Issue by mxdubois** [raisemarketplace/redux-loop#26]
Hey. I'm trying out the library and the first issue I've encountered is that some actions produce effects and others don't, which means my reducer returns inconsistent structures. I'd like to be able to normalize that for testing.

I see you've solved this problem internally with `liftReducer` / `liftState`. It'd be great to have access to those in my application/tests.

For my own sanity, I could see myself normalizing them upfront like this:

```javascript
import { Effects, loop, liftReducer: liftReducerToLooper } from 'redux-loop';
// ...
export const reducer = liftReducerToLooper(createReducer({
  [FOO_ACTION]: (state, action) => {
    return {...state, foo: 'bar' };
  },
  [EFFECTING_ACTION]: (state, action) => { 
     return loop(state, Effects.promise(someFn));
  },
}, initialState));
```

What do you think? I know there's always a tension between minimal surface area and convenience.

**Comment by mxdubois** [comment]
Actually, upon second glance, it looks like I actually want something more like `compose(liftState, myReducer)` since `liftReducer` changes the input as well as the output.

**Comment by lukewestby** [comment]
Sure, we can expose `liftState`. Do you want to submit a PR?

**Comment by lukewestby** [comment]
Included in v2.0.0!

**Comment by mxdubois** [comment]
thanks @lukewestby! sorry I didn't get around to a PR.
